R/setup.R

```diff
@@ -0,0 +1,55 @@
+
+#' Create a JSON config file
+#'
+#' @param apikey An \emph{Access Token} provided by Pushbullet (see details). If not
+#'   provided in the function call, the user will be prompted to enter one.
+#' @param config_file A string giving the path where the configuration file will
+#'   be written. RPushbullet will automatically attempt load
+#'   \code{~/.rpushbullet.json} (the default)
+#'
+#' @details This function writes a simple default configuration file based
+#'   on a given apikey.  It is intended to be run once to help new users setup
+#'   RPushbullet. Running multiple times without overriding the
+#'   \code{config_file} parameter will overwrite the default file. An \emph{Access
+#'   Token} may be obtained for free by logging into the Pushbullet website,
+#'   going to \url{https://www.pushbullet.com/#settings}, and clicking on
+#'   "Create Access Token".
+#'
+#' @export
+#' @importFrom RJSONIO toJSON
+#'
+#' @examples
+#' \dontrun{
+#'
+#' # Interactive mode.  Just follow the prompts.
+#' pbSetup()
+#' }
+#' @author Seth Wenchel
+pbSetup <- function(apikey=NULL, config_file="~/.rpushbullet.json"){
+  if(is.null(apikey)){
+    key <- readline("Please enter your API key (aka 'Access Token': ")
```

Comment:
Two space indent is not how the rest of the package is written though...

I am also considering moving from RJSONIO to the more modern json.  Thoughts?

Lastly, I am not sure I want/need this file.  Creating it is a one-off.  If we must, why not just document it in the wiki?